jquery-cheat-sheet.md

```diff
@@ -42,8 +44,50 @@ $(document).ready(function(){
   }
 });
 ```
+## Selecting elements
+
+
+## Modify content
+
+- `.html()`
+  - get or set the HTML contents
+    - get: no argument, know that it returns the innerHTML of the first jQuery object
+
+  ```javascript
+    $(".awesome").html()
+    // returns the innerHTML of the first element in the jQuery object
+  ```
+
+    - set: one argument that you want the html content to be
+
+  ```javascript
+    $(".awesome").html("this is awesome!")
+    // returns the innerHTML of the of all elements in the jQuery object to be "this is awesome!"
+  ```
+
+- `.text()`
+  - get or set the text contained, it will strip the HTML elements
+  - same as `.html()` only it returns the text of all jQuery objects selected for the getter portion
+
+Add an input tag to the `index.html`:
+
+```html
+<input type="text" class="search" placeholder="some text here ....">
```

Comment:
Not sure it's clear why this input tag is added... it doesn't seem to be referred to anywhere.